samples/cext-trey-research-python/README.md

```diff
@@ -0,0 +1,178 @@
+  * are any trey research consultants available who are AWS certified? (multi-parameter!)
+  * does trey research have any architects with javascript skills? (multi-parameter!)
+  * what trey research designers are working at woodgrove bank? (multi-parameter!)
+   * please charge 10 hours to woodgrove bank in trey research (POST request)
+   * please add sanjay to the contoso project for trey research (POST request with easy to forget entities, hoping to prompt the user; for now they are defaulted)
+
+Notice that each prompt mentions "trey"; this isn't necessary once you have mentioned Trey in a conversation, but it does help Copilot decide to call your plugin. This is an advantage of Declarative Copilots, where the plugin is explicitly declared and it's not necessary to establish intent to call it.
+
+If the sample files are installed and accessible to the logged-in user,
+
+   * find my hours spreadsheet and get the hours for woodgrove, then bill the client
+   * make a list of my projects, then write a summary of each based on the statement of work.
+
+## Plugin Features
+
+The sample showcases the following plugin features:
+
+  1. Declarative Copilot with branding and instructions, access to relevant SharePoint documents and the API plugin
+  1. API based plugin works with any platform that supports REST requests
+  1. Copilot will construct queries for specific data using GET requests
+  1. Copilot updates and adds data using POST requests
+  1. Multi-parameter queries to filter results
+  1. Show a confirmation card before POSTing data; capture missing parameters
+  1. Display rich adaptive cards
+
+## Setup
+
+### Prerequisites
+
+* [Python 3.11.x](https://www.python.org/downloads/)
+* [Visual Studio Code](https://code.visualstudio.com/Download)
+* [NodeJS 18.x](https://nodejs.org/en/download)
+* [Teams Toolkit extension for VS Code](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+  NOTE: If you want to build new projects of this nature, you'll need Teams Toolkit v5.6.1-alpha.039039fab.0 or newer
+* [Teams Toolkit CLI](https://learn.microsoft.com/microsoftteams/platform/toolkit/teams-toolkit-cli?pivots=version-three)
+  (`npm install -g @microsoft/teamsapp-cli`)
+* (optional) [Postman](https://www.postman.com/downloads/)
+
+### Setup instructions (one-time setup)
+
+1. Log into Teams Toolkit using the tenant where you will run the sample.
+
+1. Ensure the **Python extension** is installed in Visual Studio Code by searching for 'Python' in the Extensions view (Ctrl+Shift+X) and clicking Install.
+
+1. Press **CTRL+Shift+P** to open the command box and enter **Python: Create Environment** to create and activate your desired virtual environment.
+
+1. If your project doesn't yet have a file **env/.env.local.user**, then create one by copying **env/.env.local.user.sample**. If you do have such a file, ensure it includes these lines.
```

Comment:
This line isn't needed, the `file/createOrUpdateEnvironmentFile` task in `teamsapp.local.yml` will be executed when the developer starts a debug session (F5).